swift/apple/FirezoneNetworkExtension/PacketTunnelProvider.swift

```diff
@@ -46,18 +46,7 @@ class PacketTunnelProvider: NEPacketTunnelProvider {
       return
     }
 
-    let providerConfig = (protocolConfiguration as? NETunnelProviderProtocol)?.providerConfiguration
-
-    guard let connlibLogFilter = providerConfig?[TunnelProviderKeys.keyConnlibLogFilter] as? String
```

Comment:
We already validated this on save. If the user tampered with it, that's their fault.